src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.d.ts

```diff
@@ -51,6 +51,7 @@ type UniversalNavbarExpandedProps = {
     NAVLINKS: NavLink[]
     /** Estimate copy experiment on optimizely */
     estimateExperiment?: boolean
+    singleCta?: string
```

Comment:
```suggestion
    singleCta?: { href: string; title: string }
```

linting may be off if you accept this change, idk 😄

Reply:
Will take a look!